physics/rigid_motion.hpp

```diff
@@ -76,10 +78,35 @@ RigidMotion<FromFrame, ToFrame> operator*(
     RigidMotion<ThroughFrame, ToFrame> const& left,
     RigidMotion<FromFrame, ThroughFrame> const& right);
 
+// A |RigidTransformation|, its first derivative (a |RigidMotion|), and its
```

Comment:
This comment is a bit misleading, there is no `RigidTransformation` *per se* (although you can get it from the `RigidMotion`).